### ISSUE TITLE
Log levels do not apply to the static Logger

### ISSUE THREAD

**Issue by phils-hub** [nestjs/docs.nestjs.com#1617]
## Current behavior
Setting the log level for the standard Logger does not apply that level to the static instance. Consequently, it is not possible to define a global log level that applies to all calls to the standard logger.

## Input Code
```ts
// main.ts
const app = await NestFactory.createApplicationContext(
    AppModule,
    { 
      logger: (_ => { return ["error", "warn"] })
    }
)

// any other class
import { Injectable, Logger } from '@nestjs/common'

@Injectable()
export class AppService { 
  start(): void {
    Logger.log(`${AppService.name} has started.`, AppService.name)    // this will be displayed
  }
}
```

## Expected behavior
The expected behaviour is that the log level is applied to the static Logger as well.

## Possible Solution
Include a log level check in the static Logger log functions.

## Environment

<pre><code>
Nest version: 7.4.2 and the most recent version also.
 </code></pre>

**Comment by Tony133** [comment]
I would say this problem could be solved, right? as, this will be fixed in Nest v8.0.0 ♥️ 🚀

**Comment by kamilmysliwiec** [comment]
Correct @Tony133, thanks for pointing that out!
